### DISCUSSION TITLE
Update layout for "Create an invoice" page (and other pages?)

### DISCUSSION THREAD

**Discussion by bolatovumar** [btcpayserver/btcpayserver#2687] [General]
I always found it weird how "Create an invoice" page form (and other page forms) take up the width of the entire screen when they could easily be split into two columns (or three?) and take up a lot less space reducing the need to scroll and look neater.

Compare this:

![Screen Shot 2021-07-08 at 3 58 06 PM](https://user-images.githubusercontent.com/1934678/125000640-b325f680-e005-11eb-8a13-722628b5984c.png)

To what we currently use:

![Screen Shot 2021-07-08 at 3 58 51 PM](https://user-images.githubusercontent.com/1934678/125000642-b3be8d00-e005-11eb-9781-191a99942358.png)

**Comment by pavlenex** [comment]
Quite honestly, we should just hide the non-required information.
For example
- POS Data ()
- Notification URL
- Item description

And even item descriptions could be hidden and show on expansion, this way we can simplify the centered page, even more, simplify it or even keep it in a single column.

We've discussed some of the ways to "hide" the advanced fields [here](https://github.com/btcpayserver/btcpayserver/pull/2603#pullrequestreview-697984211)

Also some mocks on layouts (centered) posted here https://github.com/btcpayserver/btcpayserver/discussions/2360

Can we somehow combine all these ideas we have @bolatovumar ?

Ping @dstrukt for feedback.

**Reply by bolatovumar** [reply]
@pavlenex yeah, I thought about separating the required fields from the non-required ones. We could have an upper section with just the required fields and a lower section with "advanced" fields which could be hidden by default (or not). I think hiding them would increase the number of clicks users have to make as you are likely to use one or more of the optional fields (like store, for example).
